mesh/mesh.hpp

```diff
@@ -1330,38 +1334,46 @@ class Mesh
       /// @brief Return true if the face is a conforming face.
       bool IsConforming() const
       {
-         return conformity==Mesh::FaceConformity::Conforming;
+         return elem_1_conformity==Mesh::FaceConformity::Conforming &&
+                elem_2_conformity==Mesh::FaceConformity::Conforming;
+      }
+
+      /// @brief Return true if the face is a non-conforming fine face.
+      bool IsNonConformingFine() const
```

Comment:
This is not a good name for this method. If I understand this correctly, this returns `true` if and only if the face is fine nonconforming face AND side 1 is the fine element (which can only happen if the face is shared).

Reply:
I think this is the exact definition in the documentation above:
```
By extension, we name a face a non-conforming coarse or fine face,
if the face belonging to elem1 is a non-conforming coarse of fine face
respectively.
```

Reply:
The terms "non-conforming fine face" (aka "non-conforming slave face") and "non-conforming coarse face" (aka "non-conforming master face") are already well defined and used in other places in MFEM. It is better to not introduce a different meaning here.

Reply:
The whole point, CF PR description, is that "non-conforming fine face" is not the same as "non-conforming slave face", and likewise for "non-conforming coarse face" and "non-conforming master face".
Besides, master and slave are not documented as far as I know!

Reply:
Assigning different meanings to "non-conforming fine face" and "non-conforming slave face" will be quite confusing I think; similarly with "non-conforming coarse face" and "non-conforming master face".

Reply:
One possible name for the method that does not create ambiguity is `HasFineSide1()`.